docs/kb/simapp.md

```diff
@@ -0,0 +1,56 @@
+---
+order: 13
+description: Chain Simulation
+---
+
+# Chain Simulation
+
+The simulator can help you test different scenarios for your chain, simulating the messages, blocks, and accounts. Starport scaffold a base simulation for each module and also dummy simulation methods for each scaffold message.
```

Comment:
```suggestion
The simulator can help you test different scenarios for your chain, simulating the messages, blocks, and accounts. Starport scaffolds the template to perform simulation testing in each module and also boilerplate simulation methods for each scaffolded message.
```

Reply:
> different scenarios

`different scenarios` sounds like it's scenarios predefined by the developer. I think we should precise it is randomized inputs and it is used for benchmark and fuzz testing as well